src/views/EventsAttended/EventsAttended.module.scss

```diff
@@ -0,0 +1,9 @@
+.eventsAttended {
+  &_title {
+    color: var(--mui-palette-neutral-contrastText);
+  }
+  &_button {
+    color: var(--mui-palette-secondary-main);
+    background-color: var(--mui-palette-primary-contrastText);
+  }
```

Comment:
Please use the `color` prop instead.

Reply:
Apparently this button class isn't even being used, so I'll remove it.